functions-methods.js

```diff
@@ -12,14 +52,57 @@
 
 
 
+
+function getEmailDomain(werknemers) {
+
+    for (let i = 0; i < werknemers.length; i++) {
+        const name = werknemers[i].name;
+        const email = werknemers[i].email;
+        const domain = email.split('@')[1];
+        console.log( "naam van werknemer = "+ name + " Met het email domein " + domain);
+    }
+
+}
+
+getEmailDomain(docenten);
+console.log("\n \n")
+
+
+
+
 /* Opdracht  2 */
+
 // Schrijf een functie genaamd typeOfEmail, die een emailadres verwacht. De functie checkt of het emailadres een novi domein heeft (medewerker), een novi-education domein (student), of extern domein (zoals gmail of outlook)
 // ---- Verwachte uitkomsten:
 // typeOfEmail("n.eeken@novi-education.nl") geeft "Student"
 // typeOfEmail("t.mellink@novi.nl") geeft geeft "Medewerker"
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
 
+function typeOfEmail(werknemers) {
+
+    for (let i = 0; i < werknemers.length; i++) {
+
+        if (werknemers[i].email.split('@')[1] === 'novi-education.nl'){
+            console.log("Student");
+        }
+        else if (werknemers[i].email.split('@')[1] !== 'novi.nl') {
+            console.log("Medewerker");
+            continue;
+        }
+        else if (werknemers[i].email.split('@')[1] !== 'outlook.com') {
+            console.log("Extern");
+            continue;
+        } else
+        console.log("Extern");
```

Comment:
Ik zie `werknemers[i].email.split('@')[1]` herhaald worden, dus dit kun je lostrekken naar een variabele:
```suggestion

        const domain = werknemers[i].email.split('@')[1];
        
        switch (domain) {
            case 'novi-education.nl':
                console.log("Student");
                break;
            case 'novi.nl':
                console.log("Medewerker");
                break;
            default:
                console.log("Extern");
                break;
        }
```